wp-content/plugins/duracelltomi-google-tag-manager/integration/amp.php

```
@@ -0,0 +1,167 @@
+	// run once.
+	if ( $gtm4wp_amp_bodyinjected ) {
+		return false;
+	}
+
+	$gtm4wp_amp_bodyinjected = true;
+	// Check dataLayer is loaded from the plugin.
+	if ( ! empty( $gtm4wp_datalayer_json ) ) {
+
+		// Builds a list of GTM id's.
+		$gtm4wp_ampids     = explode( ',', $gtm4wp_options[ GTM4WP_OPTION_INTEGRATE_AMPID ] );
+		$gtm4wp_ampid_list = array();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$gtm4wp_ampid_list'. (undefined)

(UnusedLocalVariable)